python/ray/_private/usage/usage_lib.py

```diff
@@ -144,6 +148,16 @@ def _generate_cluster_metadata():
     return metadata
 
 
+def print_usage_stats_heads_up_message() -> None:
+    if _usage_stats_prompt_disabled() or _usage_stats_enabled():
+        return
+
+    print(
+        "Usage stats collection will be enabled by default in the next release. "
+        "See https://docs.google.com/document/d/1ZT-l9YbGHh-iWRUC91jS-ssQ5Qe2UQ43Lsoc1edCalc/edit#heading=h.17dss3b9evbj for more details"
```

Comment:
1. Do we want to put some color using colorama?
2. Do we want to remember that the heads-up is shown (e.g. in ~/.ray/config.json) so we only show it once on a machine, or it's overkill.